enhancements/network/host-port-registry.md

```diff
@@ -23,35 +23,71 @@ This document serves as the authoritative list of port assignments.
 
 ## Background
 
-Ports 9000-9999 are available for general use by system services. We require that
-customers open this range [and several others](https://github.com/openshift/openshift-docs/blob/master/modules/installation-network-user-infra.adoc)
-between nodes.
+Host-network ports are a shared resource in an OCP cluster, and pods
+running on the host network must coordinate their use to avoid
+conflicts.
 
-The user-facing documentation, informed by the installer's
-terraform templates, is the authoritative source for usable port ranges.
+OCP reserves host ports 9000-9999 (both TCP and UDP) for use by system
+services that communicate between nodes. Ports 29000-29999 (both TCP
+and UDP) are reserved for use by system services that listen on
+(host-network) localhost only. (OCP does not reserve any SCTP ports.)
 
-These ports are used by certain services that, for whatever reason, must run
-in the host network (or use host-port forwarding).
+Additionally, some services that have standard port assignments run on
+ports outside this range, either for historical reasons, or to allow
+them to be more easily recognized by network debugging and monitoring
+tools.
 
-The Kubernetes scheduler is aware of host ports, and will fail to schedule pods
-on a node where there would be a port conflict.
+Other than the reserved ranges and the other ports listed below, host
+ports on worker nodes are available for use by customers. (But all
+host ports on masters are reserved for OCP use.)
```

Comment:
Do we need to address what happens on single-node or compact clusters where control plane and workers are the same host?